Makefile

```diff
@@ -220,7 +220,7 @@ test/instrumented-sample-app/certs/cert.pem test/instrumented-sample-app/certs/k
 .PHONY: test-e2e
 test-e2e: KUBECONFIG?=$(HOME)/.kube/config
 test-e2e: test/instrumented-sample-app/certs/cert.pem test/instrumented-sample-app/certs/key.pem
-	go test -timeout 55m -v ./test/e2e/ $(TEST_RUN_ARGS) --kubeconfig=$(KUBECONFIG) --operator-image=$(REPO):$(TAG) -count=1
+	go test -timeout 45m -v ./test/e2e/ $(TEST_RUN_ARGS) --kubeconfig=$(KUBECONFIG) --operator-image=$(REPO):$(TAG) -count=1
```

Comment:
Why are we decreasing the timeout as well?

Reply:
That's an artifact from testing. We should rather increase timeout now as running full test suite can take more time. I will revert this and we can increase it later :)